src/mlpack/tests/main_tests/decision_stump_test.cpp

```diff
@@ -249,7 +248,7 @@ TEST_CASE_METHOD(DecisionStumpTestFixture, "DecisionStumpTrainingVerTest",
 
   // Input pre-trained model.
   SetInputParam("input_model",
-                std::move(IO::GetParam<DSModel*>("output_model")));
+                std::move(IO::GetParam<DSModel *>("output_model")));
```

Comment:
I don't see any reason for the changes in this file either; can you revert them please?